packages/nimble-blazor/NimbleBlazor/wwwroot/NimbleBlazor.lib.module.js

```diff
@@ -99,41 +111,46 @@ export function afterStarted(Blazor) {
     });
 }
 
-window.NimbleBlazor = {
-    Dialog: {
-        show: async function (dialogReference) {
-            const reason = await dialogReference.show();
-            return reason === window.customElements.get('nimble-dialog').UserDismissed;
-        },
-        close: function (dialogReference) {
-            dialogReference.close();
-        }
-    },
-    Drawer: {
-        show: async function (drawerReference) {
-            const reason = await drawerReference.show();
-            return reason === window.customElements.get('nimble-drawer').UserDismissed;
-        },
-        close: function (drawerReference) {
-            drawerReference.close();
-        }
-    },
-    Table: {
-        setData: async function (tableReference, data) {
-            const dataObject = JSON.parse(data);
-            await tableReference.setData(dataObject);
-        },
-        getSelectedRecordIds: async function (tableReference) {
-            return tableReference.getSelectedRecordIds();
-        },
-        setSelectedRecordIds: async function (tableReference, selectedRecordIds) {
-            await tableReference.setSelectedRecordIds(selectedRecordIds);
+if (window.NimbleBlazor) {
+    console.warn('Attempting to initialize NimbleBlazor multiple times!'); // eslint-disable-line
+} else {
+    window.NimbleBlazor = {
```

Comment:
Could be simplified to:

```js
if (window.NimbleBlazor) {
    console.warn('Attempting to initialize NimbleBlazor multiple times!'); // eslint-disable-line
}

window.NimbleBlazor = window.NimbleBlazor ?? {
  // existing stuff
};
```

Looks like the `nimble-blazor/.eslintrc.js` needs to specify `parserOptions` for nullish coalescing (which it should have been doing anyway):

```js
module.exports = {
    root: true,
    extends: [
        '@ni/eslint-config-javascript'
    ],
    parserOptions: {
        ecmaVersion: 2020
    },
    overrides: [
 ///
```